### ISSUE TITLE
508-defect-2 [COGNITION]: Expiration alert should be placed at the top of the page and should include next step guidance

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24640]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

Important alerts that determine if someone can continue in a flow or not should be placed at the top of a page to save folks the time of having to scroll through content. This will help everyone, but especially users with cognitive disabilities or screen readers. 

Currently, the expiration alert, which is arguably the most important bit of information on the page is located at the very bottom of the DOM. Consider a beginner screen reader user who may navigate line by line listening to the content only to realize that their questionnaire has expired at the very end of the page.

https://user-images.githubusercontent.com/14154792/118050621-9fd01500-b34d-11eb-9b52-6667f6d86e5a.mov

## Acceptance Criteria

- [ ] Expiration alert is placed at the top of the page

## Proposed Solution 

Place the alert at the top of the page and provide it a `h2` for additional screen reader visibility. Content should be provided as well explaining how a user can continue or what to do next. **This should not use the yellow background style, [use the alert headline instead](https://design.va.gov/storybook/?path=/docs/components-va-alert--warning#warning)**

<img width="1375" alt="Screen Shot 2021-05-12 at 6 18 52 PM" src="https://user-images.githubusercontent.com/14154792/118051293-bcb91800-b34e-11eb-9b1b-5de05c7ffd44.png">

**Comment by mdewey** [comment]
Hey Josh! 
 
I want to provide some context around this. This expiration message is an anomaly with the test data. This will never actually happen. The test data that was provided is not 100% reflective of how the end system will behave. The expired message should never actually show. 

To get into the weeds, we are pulling appointments from Lighthouse. Each appointment gets a status that is related to a few different factors. We are looking for appointments that have `booked` or `arrived` status. One of the factors is the start time of the appointment. For this appointment, the date in the past but the status still says it's in the future.  In our staging environment, we do not have the ability to create or modify existing appointments, so the data we are pulling is synthetic data in the Lighthouse staging env. In production, Lighthouse is actually using the data about the appointment to create the status. This means that the scenario that a user has a `booked` appointment in the past would never actually happen.  

@joshkimux  Since this scenario should never happen, what do you recommend we do with this ticket?

**Comment by joshkimux** [comment]
Awesome context! If the expiration message never shows, we can close this ticket out 😄
